test/overlapped-checker/main_win.c

```diff
@@ -0,0 +1,67 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <windows.h>
+
+char buf[256];
+DWORD read_count = 0, write_count = 0;
```

Comment:
Assignment isn't really necessary, they're initialized to 0 anyway because they're in the bss section.

Style note: prefer one definition per line:
```c
DWORD read_count;
DWORD write_count;
```